azure-pipelines.yml

```diff
@@ -3,10 +3,14 @@ resources:
     - repository: templates
       type: github
       name: NSLS-II/profile-collection-ci
-      ref: refs/heads/collection-2020-2.0rc9
+      ref: refs/heads/main
       endpoint: github
 
 jobs:
-- template: azure-linux.yml@templates  # Template reference
-  parameters:
-    beamline_acronym: BMM
+  - template: collection-2021-1.0.yml@templates
+    parameters:
+      beamline_acronym: BMM
+  - template: collection-2021-1.2.yml@templates
+    parameters:
+      beamline_acronym: BMM
```

Comment:
These two templates trigger 2 separate CI runs (with `collection-2021-1.0` and `collection-2021-1.2` conda envs):
![image](https://user-images.githubusercontent.com/13209176/112203865-5f73d680-8be9-11eb-8ec2-a034c767df3f.png)